sample-lnd.conf

```diff
@@ -846,6 +846,22 @@ litecoin.node=ltcd
 ; value must be >= 1m.
 ; healthcheck.diskspace.interval=6h
 
+; The number of times we should attempt to check for certificate expiration before
```

Comment:
In addition to adding these values to sample config, it would be nice to set some sane [default values](https://github.com/lightningnetwork/lnd/blob/master/config.go#L444) for this health check. Even if the default is off by default, it could be helpful to have the interval/backoff set.